Aptoide/appcoins-wallet-android
# [feature/APPC-1162] Merge APPC and APPC-C

**What does this PR do?**

 This PR aims to merge both payment methods in a single one. After this PR the user is presenter with a appc/appc-c payment method and then navigates to another view showing both in separate.

**Database changed?**

 No

**Where should the reviewer start?**

- MergedAppcoinsFragment.kt
- PaymentMethodsFragment.java
- merged_appcoins_layout.xml

**How should this be manually tested?**

-On the PaymentMethodsView:
--Check if button is set to Next when the merge method is selected and buy or donate in the other cases;
--Check if when the users has no appc or appc-c if the method shows as disable
-On the MergedAppcoinsView:
-Go with only appc and then only appc-c and check if one of the method is disable (both portrait and landscape
-Go with both methods enabled and check if selection is working correctly and check if the layout changes correctly on landscape (bonus)
-Make a payment with each of the methods at least once
-Make a payment with one of the methods on donation, oil and freeze
-Press back both on button and on device and check if it navigates back correctly 
-Press back on device immediately after pressing back on MergedAppcoinsView

**What are the relevant tickets?**

https://app.zeplin.io/project/5ba253dfbf8fc95444761ac4/screen/5d175198489aac5700a262c2
https://aptoide.atlassian.net/browse/APPC-1162


**Code Review Checklist**
